release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1144,16 +1207,52 @@ module openconfig-evpn {
     }
   }
 
-  grouping evpn-endpoint-vni-state {
+  grouping evpn-endpoint-vni-config {
     description
-      "Grouping for L2VNI and L3VNI state information learned on the
-      local VXLAN Tunnel End Point from remote VTEPs";
+      "Grouping for L2VNI and L3VNI configuration parameters";
 
     leaf vni {
       type oc-evpn-types:evi-id;
       description "L2VNI or L3VNI Identifier";
     }
 
+    leaf vni-type {
+      type enumeration {
+        enum L2 {
+          description
+            "This is a Layer 2 service virtual network identifier (L2VNI)
+            that is used for communication within the same subnet or
+            broadcast domain";
+        }
+        enum L3 {
+          description
+            "This is a Layer 3 service virtual network identifier (L3VNI)
+            or VRF VNI that is used for communication between subnets";
+        }
+      }
+      description "The type of virtual network identfier";
+    }
+
+    leaf bridge-domain {
+      type uint32;
+      description
+        "This reflects the configured VLAN or Bridge Domain that maps to this
+        L2VNI in the VXLAN fabric";
+    }
+
+    leaf l3-vrf-name {
+      type string;
+      description
+        "This refects the configured VRF instance that maps to this L3VNI
+        that is used for routing between subnets in the VXLAN fabric";
+    }
+  }
```

Comment:
Following precedent in OC for referring to a network instance:

```suggestion
    leaf network-instance {
          type oc-ni:network-instance-ref;
      description
        "This refects the configured VRF instance that maps to this L3VNI
        that is used for routing between subnets in the VXLAN fabric";
    }
  }
```

Reply:
The intent here was to match the existing fields in the `endpoint-vnis` state container, both to match the existing model, and also so the common parts could be factored out into a common grouping.  If we wanted to make this different, we'd either have to have a difference in structure/data type between `endpoint-vnis` and `local-endpoint-vnis`, or we'd have to change `endpoint-vnis` in a non-backwards compatible way.

Reply:
I see, ok.